content/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -1,39 +1,51 @@
 ---
 title: Circuit Breaking
-description: This task demonstrates the circuit-breaking capability for resilient applications
+description: This task shows you how to configure circuit breaking for connections, requests, and outlier detection.
```

Comment:
the extra space at the end of this line is not needed.

Reply:
Are you referring to the space after "connections,"? The line is wrapping -- I don't think I can change the way these labels are formatted it or it fails the linter. I could be mistaken though.